### PR TITLE
[mkcal] Set event occurrence times to recurring event alarms

### PR DESCRIPTION
**Pull request by pvuorela**
Was getting times from the series start.

### CODE REVIEW

File: src/extendedstorage.cpp [hunk 1]
```diff
@@ -671,10 +672,18 @@ void ExtendedStorage::setAlarms( const Incidence::Ptr &incidence )
...
+      if (incidence->recurs()) {
+        // assuming alarms not later than event start
+        eventStart = incidence->recurrence()->getNextDateTime( alarmTime.addSecs(-60) );
+      } else {
+        eventStart = incidence->dtStart();
+      }
+      e.setAttribute( "time", eventStart.toString() );
+      e.setAttribute( "startDate", eventStart.toString() );
+      if (incidence->endDateForStart( eventStart).isValid() ) {
+        e.setAttribute( "endDate", incidence->endDateForStart( eventStart ).toString() );
```
Comment (pgerdt) [inline src/extendedstorage.cpp]: Inconsistent formatting
* line 685: if (xx ), most format either if (xx) or if ( xx ) to be in line with the rest of the file, the latter seems to be used more.
* line 685 uses ( xx).xx() and 686 ( xx ).xx(). The latter seems to be used more in this file.

Both bullet points applies to the duplicated code starting from line 941 or so.

**Comment by pgerdt** [comment]
LGTM, only a minor code formatting issue caught my eye.

File: src/extendedstorage.cpp (hunk 1 above)
Comment (pvuorela) [inline src/extendedstorage.cpp]: This was actually old code, but while at it, I'll change that and above recurs() to follow ( x ) as being more common in mkcal.

**Comment by pvuorela** [comment]
Adjusted whitespace.

**Pull request merged**